src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -155,6 +155,12 @@ class FrontendGenImpl {
       auto attr = node.attribute(i);
       attributes.push_back(convertOnnxAttributeProtoToMlirNamedAttribute(attr));
     }
+
+    // If the node has a name, then import it.
+    if (node.has_name()) {
+      attributes.push_back(
+          builder_.getNamedAttr("name", builder_.getStringAttr(node.name())));
+    }
```

Comment:
@tjingrant

Reply:
It is OK to use "name". But a more special name for this attribute may be better. At least let's document this choice.

Reply:
Just so I know what's going on - would you like me to add a comment here or should that documentation go elsewhere?

Reply:
Could you change "name" to "onnx_node_name"?

Reply:
Sure, no problem.